src/lib/seam/components/SupportedDeviceTable/HiddenDevicesOverlay.tsx

```diff
@@ -0,0 +1,11 @@
+interface HiddenDevicesOverlayProps {
+  isShowing: boolean
```

Comment:
Maybe just `visible` or `hidden`?

Reply:
Updated in https://github.com/seamapi/react/pull/251/commits/0ef777243471d09f93b3e77fb6942603aee730b1